pkg/destroy/alibabacloud/alibabacloud.go

```diff
@@ -323,7 +323,7 @@ func (o *ClusterUninstaller) deleteResourceGroup(logger logrus.FieldLogger) (err
 	logger = logger.WithField("name", resourceGroupName)
 	logger.Debug("Searching resource groups")
 
-	response, err := o.listResourceGroups()
+	response, err := o.listResourceGroups(resourceGroupName)
```

Comment:
With this change, the subsequent loop through the list can be changed to just a check for `response.TotalCount > 0` and getting the resource group ID from the first element in the list.

Reply:
refer to previous comment